kratos/solving_strategies/schemes/residual_based_bossak_displacement_scheme.hpp

```diff
@@ -1,1151 +0,0 @@
-//    |  /           |
```

Comment:
why are u removing this file?????

Reply:
I am changing the name for consistency *.h instead of *.hpp

Reply:
i am not in favor of changing filenames without a good reason. Are u sure it is not included in other places within the code base?

Reply:
in any case, even if i agreed about changing the name (and i don't as i don't see aesthetics as a good reason), the way would be to move it through git, not to create a new file and delete the old one.

having said this, if people in the @KratosMultiphysics/technical-committee back you on the name change, i'll remove my block on this

Reply:
I changed everywhere where is included, i change the name, sometines git recognizes the name change, and this time it detected like I removed and added a file

Reply:
@RiccardoRossi I don't have a strong opinion about the name either way, but I agree that history should be kept whenever possible. Is there some way to ensure that history is preserved in cases such as this? Maybe do the changes on the old file and then `git mv` to the new name?